content/source/downloads.html.erb

```diff
@@ -74,4 +74,22 @@ show_notification: false
       </a>
     </div>
   </div>
+
+  <div class='cloud-learn-cta'>
+    <div class='top'>
+      <h3>Getting Started</h3>
+      <p>Terraform is distributed as a single binary. Install Terraform by unzipping it and moving it to a directory included in your system's <code>PATH</code>.</p>
```

Comment:
```suggestion
      <p>Terraform is distributed as a binary or as a package for each of the major operating systems and multiple Linux distributions. See the tutorials below which include instructions for your operating system.</p>
```

Terraform is now available in many ways and most people choose to use a package. This changes the sentence to reflect that.